packages/typescript-estree/src/convert.ts

```diff
@@ -1596,20 +1596,17 @@ export class Converter {
             expressions: []
           });
 
-          const left = this.convertChild(node.left),
-            right = this.convertChild(node.right);
-
-          if (left.type === AST_NODE_TYPES.SequenceExpression) {
+          const left = this.convertChild(node.left);
+          if (
+            left.type === AST_NODE_TYPES.SequenceExpression &&
+            node.left.kind !== SyntaxKind.ParenthesizedExpression
+          ) {
             result.expressions = result.expressions.concat(left.expressions);
           } else {
             result.expressions.push(left);
           }
 
-          if (right.type === AST_NODE_TYPES.SequenceExpression) {
```

Comment:
right node will never be `SequenceExpression` inside `ts.ParenthesizedExpression`